### PR TITLE
Attempt at fixing sub flapping test

### PR DESCRIPTION
**Pull request by Williangalvani**

### CODE REVIEW

File: Tools/autotest/vehicle_test_suite.py [hunk 1]
```diff
@@ -4460,7 +4460,7 @@ def TestLogDownloadMAVProxyNetwork(self, upload_logs=False):
             self.mavproxy_load_module(mavproxy, 'log')
             self.wait_heartbeat()
             mavproxy.send("log list\n")
-            mavproxy.expect("numLogs")
+            mavproxy.expect("numLogs", timeout=120)
```
Comment (peterbarker) [inline Tools/autotest/vehicle_test_suite.py:4463]: This is probably not going to work.

UDPMulticast and UDPBroadcast are lossy.  Neither MAVProxy nor the autotest suite have retry logic to account for that.

I added a few notes over here (which should probably be on Buzz's issue): https://github.com/ArduPilot/ardupilot/pull/26371

Comment (Williangalvani) [reply Tools/autotest/vehicle_test_suite.py:4463]: Oh i see. fair enough. I'll close this then. Why does it look like it is only happening with Sub, though?

Comment (Williangalvani) [reply Tools/autotest/vehicle_test_suite.py:4463]: oh, I just remembered why I made this:
https://github.com/ArduPilot/ardupilot/actions/runs/8178212379/job/22362301977#step:7:19363

it does look like a log list comes through after the timeout.

Comment (Williangalvani) [reply Tools/autotest/vehicle_test_suite.py:4463]: I guess there's an extra message from mavproxy  saying the total number of logs? that would make sense.

Comment (peterbarker) [reply Tools/autotest/vehicle_test_suite.py:4463]: > Oh i see. fair enough. I'll close this then. Why does it look like it is only happening with Sub, though?

The sub test suite is where the new tests for log-download-over multicast were put....

**Pull request closed without merging**